ui/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I got this when compiling

```
info  - Disabled SWC as replacement for Babel because of custom Babel configuration ".babelrc" https://nextjs.org/docs/messages/swc-disabled
info  - `compiler` options in `next.config.js` will be ignored while using Babel https://next.js.org/docs/messages/ignored-compiler-options
info  - Using external babel configuration from /home/me/prj/plato/cobudget/ui/.babelrc
```

which makes us lose the styled components optimization, and I think eventually this will be deprecated. Can we avoid adding this file?

Reply:
In the discussions in the next.js repo there seem to be a bunch of users requesting formatjs support, so that will probably get in eventually. We can keep this file for now, but we should re-add the styled-components config that we had before into it so we don't miss that optimization https://github.com/cobudget/cobudget/commit/7e66cd7f50595250746ae7bea05189e0928c9bae

Reply:
Alright, Ill keep checking